applications/matter_weather_station/Kconfig.sysbuild

```diff
@@ -4,18 +4,24 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+#### Radio core
 config NRF_DEFAULT_IPC_RADIO
-	default y if BOARD_THINGY53_NRF5340_CPUAPP
+	default y
+
+config NRF_DEFAULT_BLUETOOTH
+	default y if WIFI_NRF700X
+
+config NETCORE_IPC_RADIO_BT_HCI_IPC
```

Comment:
Shouldn't it depend on the board selection?

Reply:
I mean that BT_HCI_IPC should be enabled only for the Thingy+nRF7002EB case, right? So why it is enabled always?